### PR TITLE
Add git autoupdate config in spinejs

### PR DESCRIPTION
**Pull request by Piicksarn**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Wrong config, **again**.

**Comment by Piicksarn** [comment]
it has been corrected.

**Comment by PeterDaveHello** [comment]
No you **didn't**.

**Comment by Piicksarn** [comment]
have it corrected~thank you~

**Comment by PeterDaveHello** [comment]
Should not use `**/*` in non-dist folder.

**Comment by Piicksarn** [comment]
okay

**Comment by PeterDaveHello** [comment]
... same same

**Comment by Piicksarn** [comment]
@PeterDaveHello is this one right?

**Comment by PeterDaveHello** [comment]
Not at all, you should not use wildcard in non-dist folder, especially its prefix is not the lib's name.

**Comment by Piicksarn** [comment]
sorry....
is it ok now  :pensive:

File: ajax/libs/spinejs/package.json [hunk 1]
```diff
@@ -25,5 +25,33 @@
...
     }
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/spine/spine.git",
+    "basePath": "lib",
+    "files": [
+      "spine.js",
+      "spine.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/spinejs/package.json]: Where is `spine.min.js`? You must be kidding.

**Comment by Piicksarn** [comment]
I think it's fine now! :dancer::dancer::dancer:
thank you !!!!

**Comment by PeterDaveHello** [comment]
You are really kidding, let's close this one and do it from scratch as a practice for you ... hop you can do it correctly next time.

**Pull request closed without merging**